fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -107,12 +107,13 @@ def self.choose_lane(ff, platform)
 
       puts(table)
 
+      fastlane_command_prefix = Helper.bundler? ? "bundle exec " : ""
```

Comment:
```ruby
"#{fastlane_command_prefix}fastlane"
```

It's not a big deal but I prefer not to see a variable interpolated and a string literal combined, like above, as I might wonder if there is a whitespace between those two. What do you think about composing all the command name here so that my small concern is resolved? 🙂 

```suggestion
      fastlane_command = Helper.bundler? ? "bundle exec fastlane" : "fastlane"
```